### PR TITLE
Added Demo for SmartMultiInput feature

### PR DESCRIPTION
**Pull request by mzinnoecker**
See [PR 1630](https://github.com/abap2UI5/abap2UI5/pull/1630)

### CODE REVIEW

**Comment by oblomov-dev** [comment]
everything merged, i think it should work now. thank you for the PR!

Unfortunately it seems that i can not test it on ABAP dev edition:
image

do you have some screenshots or a link to the documentation that i can see what it is about? maybe i can find a service in the dev edition for that as well.

**Comment by mzinnoecker** [comment]
Thanks!
Yes i will take some screenshots tomorrow :)
Essentially it enables the usage of SmartMultiInput. If annotated correctly (Consumption.ValueHelpDefinition) we have to also link the backend annotations in for  the ui5 odata model, thats why i added the parameter switchDefaultModelAnnoURI. It is required for the smartMultiInput to render the table/auto suggestions based on the backend annotations. 

In short: will take some screenshots tomorrow :)

**Comment by mzinnoecker** [comment]
hey, as promised:

Service Binding  
Product Type  
Selection  
Auto completion 
range content in backend

**Comment by oblomov-dev** [comment]
thank you for the screenshot, ah its this feature, always used it automatically with Fiori Elements, but never accessed it manually in UI5 freestyle. Very nice, thank you for figuring it out and adding it to abap2UI5.

adjusted your sample to the demo travel odata service which is available on dev editions:
Eclipse 2024-12-11 13 10 00
Google Chrome 2024-12-11 13 10 19

And renamed for consistency to:
switch_default_model_anno_uri
